src/util/dstring.h

```diff
@@ -13,6 +13,9 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "string_container.h"
 
+class serializert;
+
```

Comment:
One blank line should be enough to visually separate blocks. Two, three or more blank lines just stretch the code unnecessarily.